server/index.ts

```diff
@@ -212,41 +213,36 @@ app.all(
 			serverBuild: getBuild(),
 		}),
 		mode: MODE,
-		// @sentry/remix needs to be updated to handle the function signature
-		build: MODE === 'production' ? await getBuild() : getBuild,
+		build: getBuild,
 	}),
 )
 
 const desiredPort = Number(process.env.PORT || 3000)
-const portToUse = await getPort({
+const portToUser = await getPort({
 	port: portNumbers(desiredPort, desiredPort + 100),
 })
 
-const server = app.listen(portToUse, () => {
+const server = app.listen(portToUser, () => {
 	const addy = server.address()
-	const portUsed =
-		desiredPort === portToUse
```

Comment:
A tine [chance](https://github.com/sindresorhus/get-port?tab=readme-ov-file#beware), but still it makes sense to only trust the OS's final decision ;)